package.json

```diff
@@ -61,7 +61,7 @@
     "@storybook/preset-typescript": "^3.0.0",
     "@types/draco3d": "^1.4.0",
     "@types/offscreencanvas": "^2019.6.4",
-    "@types/three": "^0.137.0",
+    "@types/three": "^0.141.0",
```

Comment:
This is causing CI to fail since r141 had a breaking change to some internal WebXR types. I'd keep this at 140 until we can clear a major to resolve this.